File: exercises/JavaScript/chapter_1/4_total_purchase.js
// 4. Total Purchase
// A customer in a store is purchasing five items. Design a Python program that asks for the price of each item, and then
// displays the subtotal of the sale, the amount of sales tax, and the total. Assume the sales tax is 6 percent.

const item_1 = 10
const item_2 = 12
const item_3 = 9
const item_4 = 23
const item_5 = 18

const subtotal = item_1 + item_2 + item_3 + item_4 + item_5
const sales_tax = 0.06 * subtotal
const total = subtotal + sales_tax

console.log("Subtotal:", subtotal)
console.log("Sales Tax:", sales_tax)
console.log("Total:", total)

File: exercises/Python/chapter_1 -  Basics/4_total_purchase.py
"""
4. Total Purchase
A customer in a store is purchasing five items. Design a Python program that asks for the price of each item, and then
displays the subtotal of the sale, the amount of sales tax, and the total. Assume the sales tax is 6 percent.
"""

item_1 = float(input("Price for item #1: "))
item_2 = float(input("Price for item #2: "))
item_3 = float(input("Price for item #3: "))
item_4 = float(input("Price for item #4: "))
item_5 = float(input("Price for item #5: "))

subtotal = item_1 + item_2 + item_3 + item_4 + item_5
sales_tax = 0.06 * subtotal
total = subtotal + sales_tax

print("Subtotal:", subtotal)
print("Sales Tax:", sales_tax)
print("Total:", total)
